app/assets/javascripts/miq_jqplot.js

```diff
@@ -68,6 +71,27 @@ function jqplot_redraw_charts() {
     }
 }
 
+function jqplot_pie_highligh_values(str, seriesIndex, pointIndex, plot) {
+    return plot.series[seriesIndex].data[pointIndex][0] + ': ' +
+           plot.series[seriesIndex].data[pointIndex][1];
+}
+
+function jqplot_pie_highligh(str, seriesIndex, pointIndex, plot) {
```

Comment:
Yes. I'll look into it.

Also In master we are scoping all the javascript functions under `ManageIQ.whatever` context. So I will have to touch this area soon. But we do not have the scoping in production so I have to backport this first.

Also I still have to use @himdel #4003 and axis ticks formating.